### PR TITLE
Fix(DB/creatures): UBRS stealthed mobs issues

### PR DESCRIPTION
**Pull request by patou01**
## Changes Proposed:
-  removes sneak aura that was applied to the db

## Issues Addressed:

- closes https://github.com/azerothcore/azerothcore-wotlk/issues/8352

## Tests Performed:

- built and went there, mob not stealthed anymore
- 

## How to Test the Changes:

1. .go c 137990, see the mob is not in sneak anymore. Note that it's a pat, so he can be kind of wherever between that point and the bridge to Drakky (walks along the split to the beast)
2. Aggro any of the packs that have assassins (the ones after the Rend event), and see the assassins only remove stealth on attacking.

## Known Issues and TODO List:

- [ ]
- [ ]

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

**Comment by patou01** [comment]
@azerothcore/bug-triaging would any of you know what behaviour the assassins should have? I don't see any ambush or cheap shot on their wowhead table https://classic.wowhead.com/npc=10318/blackhand-assassin#abilities;mode:normal and they don't have anything ig.

**Comment by syssneck** [comment]
The strange thing is, that I found a Classic video, where an assassin is already visible without anybody aggroing or attacking him:
https://youtu.be/awPyUAF2yU8?t=1339 (skull mob on the far left)

**Comment by patou01** [comment]
Could it be that they're only stealthed when patrolling. Or never stealthed actually?

**Comment by syssneck** [comment]
The stealthed when patrolling makes sense to me, but I really don't know, sorry. I will try to watch a few more videos of UBRS.

File: data/sql/updates/pending_db_world/rev_1635700076222913000.sql [hunk 1]
```diff
@@ -0,0 +1,11 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1635700076222913000');
+
+-- remove sneak from iron guard
+DELETE FROM `creature_addon` WHERE (`guid` IN (137990));
```
Comment (Nefertumm) [inline data/sql/updates/pending_db_world/rev_1635700076222913000.sql:4]:
```suggestion
DELETE FROM `creature_addon` WHERE `guid` = 137990;
```

File: data/sql/updates/pending_db_world/rev_1635700076222913000.sql [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+INSERT INTO `creature_addon` (`guid`, `path_id`, `mount`, `bytes1`, `bytes2`, `emote`, `isLarge`, `auras`) VALUES
+(137990, 1379900, 0, 0, 1, 0, 0, '');
+
+
+-- remove "remove sneak" from assassin, this way they drop the sneak on attacking
+UPDATE `creature_template` SET `AIName` = 'SmartAI' WHERE `entry` = 10318;
+DELETE FROM `smart_scripts` WHERE (`entryorguid` = 10318) AND (`source_type` = 0) AND (`id` IN (1));
```
Comment (Nefertumm) [inline data/sql/updates/pending_db_world/rev_1635700076222913000.sql:11]:
```suggestion
DELETE FROM `smart_scripts` WHERE `entryorguid` = 10318 AND `source_type` = 0 AND `id` = 1;
```

File: data/sql/updates/pending_db_world/rev_1635700076222913000.sql [hunk 3]
```diff
@@ -0,0 +1,11 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1635700076222913000');
+
+-- remove sneak from iron guard
+DELETE FROM `creature_addon` WHERE (`guid` IN (137990));
+INSERT INTO `creature_addon` (`guid`, `path_id`, `mount`, `bytes1`, `bytes2`, `emote`, `isLarge`, `auras`) VALUES
+(137990, 1379900, 0, 0, 1, 0, 0, '');
+
+
```
Comment (Nefertumm) [inline data/sql/updates/pending_db_world/rev_1635700076222913000.sql:8]:
```suggestion
```

**Review by Nefertumm — APPROVED** [review]
Seems ok to me, just a minor codestyle check and we're ready

**Comment by syssneck** [comment]
Tested. The patrolling guard has no sneak anymore, and the assassins remove sneak on attacks.
If they should not be stealthed at all, we could simply change it later in the SAI, so I would go for the merge.

**Pull request merged**